crypto/context.c

```diff
@@ -32,15 +32,18 @@ static int context_deinit(OPENSSL_CTX *ctx)
 }
 
 static CRYPTO_ONCE default_context_init = CRYPTO_ONCE_STATIC_INIT;
-static void do_default_context_deinit(void)
+void do_default_context_deinit(void)
 {
     context_deinit(&default_context);
 }
 DEFINE_RUN_ONCE_STATIC(do_default_context_init)
 {
-    return OPENSSL_init_crypto(0, NULL)
-        && context_init(&default_context)
-        && OPENSSL_atexit(do_default_context_deinit);
+    return context_init(&default_context)
+#ifndef FIPS_MODE
```

Comment:
Breaking a statement in the middle is kinda yucky.  Might not be worth fixing tho.

Reply:
This change was done in the first commit in this PR and then changed again in the second commit, so it no longer breaks in the middle.